lib/display/display.ts

```diff
@@ -46,48 +49,35 @@ function findDependencyLines(
   const emptyDependencySection = '';
 
   return isAllowedToShowDependencies
-    ? displayDependencies(depGraph)
+    ? displayDependencies(depGraph, fileSignaturesDetails)
```

Comment:
isn't it affecting the different commands?
like showing filePaths when they are not supposed to appear with --print-deps? while they are supposed to appear only with --print-dep-paths?

Reply:
@anthogez It looks like this code does the correct thing, or do I misunderstand something?
And we only show `depsFilePaths` if `isAllowedToShowDependenciesWithFilePaths` is evaluated to true on line 43.

Regardless, I pushed a commit as an alternative (also in this branch)

Reply:
@dekelund I took a quick look yesterday, so I've asked if we did not break any functionality

while changing
```
 if (isAllowedToShowDependenciesWithFilePaths) {
    return displayDependenciesWithFilePaths(depGraph, depsFilePaths);
}
```
 With the fn below
 
```
if (isAllowedToShowDependenciesWithFilePaths) {
    return displayDependencies(depGraph, fileSignaturesDetails, depsFilePaths);
}
```

I will take a look in details soon and run it locally